_hooks/pre-commit

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# A git hook script that runs the same checks that are executed on Travis to
+# verify your commits will not fail. Bypass it with the --no-verify option to
+# git commit
+#
+# usage: make a symlink to this file,
+# e.g., rm -rf .git/hooks/ && ln -s -f ../_hooks .git/hooks
+
+#######################################
+# Error function to display error message to end users.
+# Arguments:
+#   Error message description
+#######################################
```

Comment:
Oh we don't need a function docstring here. If we reduce it to a one-liner, it's self-explanatory.